collector/collector.go

```diff
@@ -0,0 +1,77 @@
+package collector
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"net"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/containous/traefik/cmd/traefik/anonymize"
```

Comment:
🤔 why `anonymize` is under `cmd/traefik` package ?

Reply:
(or why `collector` is in a different parent package that `anonimize` ?)

Reply:
Because before this PR it was use only by `traefik bug` command.

I can move the package `anonymize` at the root of the project.